### ISSUE TITLE
Hot reload is not working!

### ISSUE THREAD

**Issue by sahilrajput03** [solidjs/solid-refresh#14]

**Comment by LXSMNSYC** [comment]
@ryansolid please see the linked issue that I mentioned. Apparently the bug only exists in the JS template

**Comment by sahilrajput03** [comment]
> @ryansolid please see the linked issue that I mentioned. Apparently the bug only exists in the JS template

 Yes, I should have mentioned that I continued to use ts temlate and its working qute superb for me!
